src/doing/utils.py

```diff
@@ -255,3 +255,14 @@ def replace_user_aliases(text: str) -> str:
         return " ".join(words)
     else:
         return " ".join([aliases.get(word, word) for word in words])
+
+def get_current_work_item_id():
+    """
+    Retrieves current work item id from the current branchname
+    """
+    wi_id = shell_output("git branch --show-current | cut -d '_' -f 1 | sed 's/[^0-9]*//g'")
```

Comment:
`sed` has some subtle differences between OS and shells.

I'd prefer to do as much as possible in python. So something like:

```python
import re 
branch_name =  shell_output("git branch --show-current")
wi_id = re.search(r"^([0-9]*)_", branch_name).group(1)
..
```

If you're not comfortable doing this in python, let me know & I'll pick it up.

Reply:
implemented and tested your suggestion, works flawless!